scripts/test/lib_subctl_gather_test.sh

```diff
@@ -114,8 +114,8 @@ function validate_resource_files() {
 }
 
 function validate_broker_resources() {
-  validate_resource_files $submariner_broker_ns 'endpoints.submariner.io' 'Endpoint' '' 'broker'
-  validate_resource_files $submariner_broker_ns 'clusters.submariner.io' 'Cluster' '' 'broker'
-  validate_resource_files $submariner_broker_ns 'serviceimports.multicluster.x-k8s.io' 'ServiceImport' '' 'broker'
-  validate_resource_files $submariner_broker_ns 'endpointslices.discovery.k8s.io' 'EndpointSlice' '' 'broker'
+  validate_resource_files $submariner_broker_ns 'endpoints.submariner.io' 'Endpoint' '' "$1"
+  validate_resource_files $submariner_broker_ns 'clusters.submariner.io' 'Cluster' '' "$1"
+  validate_resource_files $submariner_broker_ns 'serviceimports.multicluster.x-k8s.io' 'ServiceImport' '' "$1"
+  validate_resource_files $submariner_broker_ns 'endpointslices.discovery.k8s.io' 'EndpointSlice' '' "$1"
```

Comment:
```suggestion
  validate_resource_files $submariner_broker_ns 'endpoints.submariner.io' 'Endpoint'
  validate_resource_files $submariner_broker_ns 'clusters.submariner.io' 'Cluster'
  validate_resource_files $submariner_broker_ns 'serviceimports.multicluster.x-k8s.io' 'ServiceImport'
  validate_resource_files $submariner_broker_ns 'endpointslices.discovery.k8s.io' 'EndpointSlice'
```

4th arg is "selector" which can be omitted, 5th arg defaults to `$cluster` which is set globally by `with_context`

Reply:
@skitt kind reminder :point_up:

Reply:
Ah yes, thanks!